File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int i = 1;
        int g = 0;
        while(true){
            if(i>n || i>m){
                break;
            }
            if(n%i==0 && m%i==0){
                g = i;
                i++;
            }
            else {
                i++;
            }
        }
        int[] answer = {g, n*m/g};
        return answer;
    }
}

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
        let i = 1;
        let g = 0;
        while(true){
            if(i>n || i>m){
                break;2
            }
            if(n%i === 0 && m%i === 0){
                g = i;
                i++;
            }
            else {
                i++;
            }
        }
        var answer = [g, Math.floor(n*m/g)];
    return answer;
}